packages/core/test/oracle/Voting.js

```diff
@@ -1932,6 +1933,89 @@ contract("Voting", function(accounts) {
       price2.toString()
     );
   });
+  it("Stress testing the size of ancillary data", async function() {
+    let identifier = web3.utils.utf8ToHex("stress-test");
+    let time = "1000";
+    const DATA_LIMIT = 8192;
+    let ancillaryData = web3.utils.bytesToHex(crypto.randomBytes(DATA_LIMIT));
```

Comment:
Does anyone know a better way to create random hex data?

Reply:
thats a pretty good way but i remember seeing this: https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#randomhex

Reply:
Oh this is a great find! thanks

Reply:
Done

Reply:
Yeah, we use the web3 method in a few older tests to generate random addresses :). My guess is that the web3 method is basically a simple wrapper over what you had originally haha.